### ISSUE TITLE
When I click on the Copy the File Location option, it gives an error

### ISSUE THREAD

**Issue by TRShadow33** [Flow-Launcher/Flow.Launcher#1629]
[2022-12-13.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/10214840/2022-12-13.txt)

Flow Launcher version: 1.10.0
OS Version: Microsoft Windows NT 10.0.22000.0
IntPtr Length: 8
x64: True

Python Path: 
Date: 12/13/2022 08:40:39
Exception:
System.Runtime.InteropServices.COMException (0x800401D0): OpenClipboard Başarısız (0x800401D0 (CLIPBRD_E_CANT_OPEN))
   at System.Windows.Clipboard.GetDataObjectInternal()
   at System.Windows.Clipboard.GetDataInternal(String format)
   at System.Windows.Clipboard.GetText(TextDataFormat format)
   at Flow.Launcher.ViewModel.MainViewModel.<>c__DisplayClass121_0.<ConstructQuery>b__0() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 767
   at System.Windows.Threading.Dispatcher.Invoke(Action callback, DispatcherPriority priority, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback)
   at Flow.Launcher.ViewModel.MainViewModel.ConstructQuery(String queryText, IEnumerable`1 customShortcuts, IEnumerable`1 builtInShortcuts) in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 763
   at Flow.Launcher.ViewModel.MainViewModel.QueryResults() in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 741
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Comment by jjw24** [comment]
Are you constantly getting this error or just once off? What happens if you restart flow?

**Comment by TRShadow33** [comment]
I was taking the mistake all the time. Even if I restarted the Flow, I kept getting mistakes again. Rarely, when I tried to copy the path over and over again for the same folder/file, it worked without any error. He worked once in 4 trials without error. But with the 4.10.1 update, I guess it didn't give an error. It seems like it doesn't give an error right now.

> Are you constantly getting this error or just once off? What happens if you restart flow?
